discovery-provider/plugins/network-monitoring/src/content/queries.ts

```diff
@@ -336,6 +337,7 @@ export const getSecondary1WalletBatch = async (
     FROM network_monitoring_users
     WHERE run_id = :run_id
     AND secondary1spid = :spid
+    ORDER BY user_id DESC
```

Comment:
how does adding an order by modifer make it so that users dont get fetched twice / prevent null replica set clock values? is there a filter later on?

Reply:
Without the `ORDER BY`, the SQL query was returning users in a random order so the window of users selected with `OFFSET` and `LIMIT` wasn't guaranteed to have new/unique users. Having an `ORDER BY` means we're always getting unique users every batch